README.md

```diff
@@ -32,16 +32,22 @@ $ ember exam --server
 
 The idea is that you should be able to replace `ember test` with `ember exam` and never look back.
 
-However, to get the unique features of Ember Exam (described in-depth below), you will need to load Ember Exam before your test suite runs. This is done by invoking the `loadEmberExam` function, usually from within `test-helper.js` but it can be done anywhere that executes prior to your tests loading.
+However, to get the unique features of Ember Exam (described in-depth below), you will need to import functions in `addon-test-support/load.js` to load Ember Exam, to load test manually and to set up qunit callbacks to enable `load-balance` option before your test suite runs. This is done by invoking the `loadEmberExam`, `loadTests`, and `setupQunitCallbacks` function, usually from within `test-helper.js` but it can be done anywhere that executes prior to your tests loading.
```

Comment:
I would say "you will need to import the 'emberExam' object from ember-exam/test-support/load.js, then call the three functions shown below: